docs/content/docs/tutorials/rds-example.md

```diff
@@ -256,6 +256,62 @@ spec:
 EOF
 ```
 
+## Restore a Database Snapshot
+
+You can also restore a database snapshot to a specific `DBInstance` or `DBCluster` using the ACK for RDS controller.
+
+To restore a database snapshot to a `DBInstance`, you must set the `Spec.DBSnapshotIdentifier` parameter. `Spec.DBSnapshotIdentifier` should match the identifier of an existing DBSnapshot.
```

Comment:
Looking at this again, if we use the `spec` prefix we should follow the convention from earlier in the tutorial:

`spec.dbSnapshotIdentifier`.

Reply:
I will follow the convention to remove the `Spec`here